pom.xml

```diff
@@ -279,18 +270,30 @@
                         </goals>
                         <configuration>
                             <rules>
-                                <dependencyConvergence />
+                                <dependencyConvergence/>
                             </rules>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.sonatype.plugins</groupId>
+                <artifactId>nexus-staging-maven-plugin</artifactId>
+                <version>1.6.8</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <serverId>ossrh</serverId>
+                    <nexusUrl>https://oss.sonatype.org/</nexusUrl>
```

Comment:
Shouldn't this be: https://oss.sonatype.org/service/local/staging/deploy/maven2?

Reply:
according to the guide it should be enough:
https://central.sonatype.org/publish/publish-maven/#distribution-management-and-authentication